package.json

```diff
@@ -22,7 +22,8 @@
     "@turf/circle": "^6.0.1",
     "@turf/distance": "^6.0.1",
     "@turf/helpers": "^6.1.4",
-    "@turf/length": "^6.0.2"
+    "@turf/length": "^6.0.2",
+    "@turf/turf": "^5.1.6"
```

Comment:
import the modules that are necessary, turf is a huge library which increases the build size significantly. revert the `package-lock.json` file and the `package.json` before adding the dependencies.